x-pack/plugins/security_solution/public/detections/components/rules/rule_preview/preview_histogram.tsx

```diff
@@ -53,6 +58,8 @@ const FullScreenContainer = styled.div<{ $isFullScreen: boolean }>`
 
 export const ID = 'previewHistogram';
 
+export const CHART_HEIGHT = 150;
```

Comment:
nit: I don't see any usages outside so it can an internal constant similar to the others responsible for the chart height.

`150px` looks as the most common default so it looks good to export some common constant like `DEFAULT_CHART_HEIGHT` and define custom height where it's necessary.